sound/soc/intel/common/soc-acpi-intel-cnl-match.c

```diff
@@ -27,8 +27,39 @@ struct snd_soc_acpi_mach snd_soc_acpi_intel_cnl_machines[] = {
 };
 EXPORT_SYMBOL_GPL(snd_soc_acpi_intel_cnl_machines);
 
+static const struct snd_soc_acpi_endpoint single_endpoint = {
+	.num = 0,
+	.aggregated = 0,
+	.group_position = 0,
+	.group_id = 0,
+};
```

Comment:
I know that sometimes this is done for the purpose of "explicitness," but all these are redundant. A simple `static const struct snd_soc_acpi_endpoint single_endpoint;` would do exactly the same.

Reply:
I prefer things to be explicit here.